src/site/content/en/blog/periodic-background-sync/index.md

```diff
@@ -398,43 +396,18 @@ event to use, and to trigger it as many times as you'd like.
 {% endAside %}
 
 <figure class="w-figure  w-figure--center">
-  <img class="w-screenshot" src="3-sw-panel.png" 
+  <img class="w-screenshot" src="3-sw-panel.png"
        alt="The 'Service Workers' section of the Application panel shows a 'Periodic Sync'
             text field and button." style="max-width: 90%">
 </figure>
 
-## Enabling the DevTools interface
```

Comment:
@jpmedley I'm not seeing the Periodic Background Sync section enabled by default, even in Chrome 81:

![image](https://user-images.githubusercontent.com/4713486/71033401-15d63b00-20cc-11ea-9efa-fe110f63204e.png)